### PR TITLE
fix lang and relative show view

### PR DESCRIPTION
**Pull request by chrysalag**
[Link to story in trello](https://trello.com/c/YW6DFbLP/6-as-a-developer-i-want-to-process-through-the-whole-application-and-provide-feedback-1h)

- [ ] Story under test
- [ ] Mobile works
- [ ] Seeds created
- [ ] Translated

Review by Team
- [ ] Reviewed by tnt
- [ ] Reviewed by cla
- [ ] Reviewed by vok

Review by Coach
- [x] Reviewed by sta

### CODE REVIEW

File: app/views/relatives/_show.html.slim [hunk 1]
```diff
@@ -5,6 +5,7 @@ tr
       dl.dl-horizontal
         - relative.relatives.each do |rel|
           dt= rel.full_name
-          dd
-            = l(rel.date_of_birth) if rel.date_of_birth
-            |, #{rel.relation}
+          -unless rel.full_name.blank?
```
Comment (kitsane) [inline app/views/relatives/_show.html.slim]: `- if rel.full_name.present?`

File: app/views/language_skills/_show.html.slim [hunk 2]
```diff
@@ -4,5 +4,6 @@ tr
     - if speaker.language_skills.any?
       dl.dl-horizontal
         - speaker.language_skills.each do |lang|
-          dt= lang.language_name
-          dd= t("language_level.#{lang.level}")
+          - unless lang.language_name.blank?
```
Comment (kitsane) [inline app/views/language_skills/_show.html.slim]: `- if lang.language_name.present?`

Comment (casaper) [inline app/views/language_skills/_show.html.slim]: but why can there be an empty `lang.language_name` after `speaker.language_skills.any?` was true?

File: app/views/relatives/_show.html.slim (hunk 1 above)
Comment (casaper) [inline app/views/relatives/_show.html.slim]: How can `rel.full_name` ever be `blank?` after we have `validates :first_name, :last_name, :email, presence: true` in the model?

Comment (chrysalag) [inline app/views/relatives/_show.html.slim]: @casaper Not in the relative model.

File: app/views/language_skills/_show.html.slim (hunk 2 above)
Comment (chrysalag) [inline app/views/language_skills/_show.html.slim]: @casaper Currently it is possible to select language level without selecting language.

**Comment by chrysalag** [comment]
Forgot to change the base of this branch to develop... It is not possible to re-open and edit this PR, so I'll do a new one.

**Pull request merged**